spring-cloud-aws-dynamodb/src/test/java/io/awspring/cloud/dynamodb/DynamoDbTemplateIntegrationTest.java

```diff
@@ -261,11 +458,36 @@ void dynamoDbTemplate_saveAndScanForParticularIndex_entitySuccessful() {
 		cleanUp(personEntity2.getUuid());
 	}
 
+	@Test
+	void prefixedDynamoDbTemplate_saveAndScanForParticularIndex_entitySuccessful() {
```

Comment:
I am not sure if we need all these integration tests but assuming we do, lets create a nested class with JUnit 5 `@Nested` and move them all there

Reply:
@maciejwalkowiak Yeah, I kind of agree that having  duplicated test cases - one for the prefixed secenario and another for the un-prefixed scenario - is just leading us to un-maintainable test suites. I will wither `nest` them or do some parametrized test to ease the pain.

Thanks for bring this up.

Reply:
@maciejwalkowiak This is done. I parametrized tests, so we have now test that cover both scenarios without any extra code; in a lighter note - "buy one get one free" :-)